releases/handbook.md

```diff
@@ -1,193 +1,71 @@
+* Week 11 - (Bug Fixing) - Discuss any potential blockers for the Release.
+* Week 12 - (Release) - Items to discuss: bug fixes required, release cut, documentation needed.
+- Manifests WG leads synchronizes and cut the release on Kubeflow Platform/Manifests.
+- The Release Manager is responsible for approving/reviewing the blog and slides announcing the release.
 
 ## People and Roles
 
-### Release Manager
-The Release Manager will be responsible for coordinating the release and taking ultimate accountability for all release tasks to be completed on time.
-
-The previous Release Manager is responsible for choosing their successor that meets the prerequisite. If multiple Release Team members meet the prerequisites to become a Release Manager, the one from a different organization than the one that drove the previous release should be preferred.
-
-**Prerequisites:**
-1. Have contributed as a release team member in two or more previous releases
-
-**Time Commitments:** This role will need to be synchronous. The Release Manager is expected to join most of the weekly meetings and coordinate with the Release Team and the Working Group leads. For the Development phase, an estimate of the work needed is 4-8 hours per week. For the next phases, the Release Manager is required to devote more time and coordinate the tasks of the release.
-
-**Responsibilities:**
-* Provide updates to the mailing list with the progress of the release
+### Release Management Team: 
+The Release Management Team is composed by: Release manager, WG leads / liasons, technical documentation lead and product manager.
+Release team liaisons will be responsible not for the communication but for contributing to the release with documentation, source code, PRs review, etc, according to their skills and motivation.
+
+### Release Manager Responsibilities: 
+- Responsible for the overall Kubeflow Release Process 
+- Promote best practices for the release and software development process.
+- Manage the communication between the teams to understand current release status and potential blockers
+- Manage the communication with the community about the status of the Release or any help/blockers needed.
+- Approve & review the blog and slides announcing the Release.
+- Provide updates to the mailing list with the progress of the release
 * Coordinate directly with the WG liaisons and leads about dates and deliverables
 * Coordinate with the Release Team Members for the progress of the release
 * Ensure the WG leads have cut the necessary GitHub branches and tags for the different phases of the release
 * Host the weekly Release Team meetings
 * Update the larger community on release status during the Kubeflow community meetings
 * Make sure the processes are being followed
-* All the responsibilities of a Release Team Member
-
-**Authority:**
```

Comment:
@juliusvonkohout  if this is not responsability of the release manager who's responsable for making these decisions?

Reply:
The goal is to keep it short an concise. Go from this angle: What else would you remove to add this? Is this an essential statement?